src/server/game/Handlers/CharacterHandler.cpp

```diff
@@ -1456,6 +1456,11 @@ void WorldSession::HandleTutorialFlag(WorldPackets::Misc::TutorialSetFlag& packe
     }
 }
 
+void WorldSession::HandleSetWarMode(WorldPackets::Character::SetWarMode& packet)
```

Comment:
Move this to MiscHandler.cpp, next to WorldSession::HandleSetPvP

And you should also check PLAYER_LOCAL_FLAG_WAR_MODE inside both `WorldSession::HandleTogglePvP` and `WorldSession::HandleSetPvP` and return without doing anything - don't want to trigger the 5 minute pvp timer rule when war mode is enabled